ReactSkia/components/RSkComponentText.h

```diff
@@ -21,7 +22,7 @@ class RSkComponentRawText final : public RSkComponent {
   void OnPaint(SkCanvas *canvas) override;
 };
 
-class RSkComponentParagraph final : public RSkComponent {
+class RSkComponentParagraph final : public RSkComponent ,public RSkTextLayoutManager{
```

Comment:
Any reasons if we should inherit instead of compositing as a member?

Reply:
Nice catch. Compositing as member is meaningful for TextComponent. Will update the change.